content/docs/guides/auth/ALTS.md

```diff
@@ -0,0 +1,246 @@
+---
+title: ALTS Authentication
+description: >
+  An overview of gRPC authentication using Application Layer Transport Security
+  (ALTS).
+---
+
+### Overview
+
+Application Layer Transport Security (ALTS) is a mutual authentication and
+transport encryption system developed by Google and used for securing RPC
+communications within Google's infrastructure. ALTS is similar in concept to
+mutual TLS but has been designed and optimized to meet the needs of Google's
+production environments. For more information about ALTS and how it works, see
+the
+[ALTS whitepaper](https://cloud.google.com/security/encryption-in-transit/application-layer-transport-security).
+
+ALTS is now available to all gRPC users, if the application runs on
+[Google Compute Engine (GCE)](https://cloud.google.com/compute/) or
+[Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine). It
+has the following features:
+
+-   Create gRPC servers & clients with ALTS as the transport security protocol.
+-   Transparent identity provisioning and key management of ALTS credentials.
+-   ALTS connections are end-to-end protected with privacy and integrity.
+-   Applications can access peer information such as the peer service account.
+-   Client authorization and server authorization support.
+-   Minimal code changes to enable ALTS.
+
+gRPC users can configure their applications to use ALTS as a transport security
+protocol with few lines of code, simply indicating the intent of using ALTS.
+gRPC ALTS is supported in C++, Java, Go, and Python.
+
+### Identity and Key Management
+
+Using ALTS transport security protocol, the identity of the gRPC application is
+the primary service account associated with the GCE VM that the application runs
+on. The service account of a GCE VM can be set or changed using
+[gCloud command](https://cloud.google.com/sdk/gcloud/reference/compute/instances/set-service-account)
+or via
+[GCP console](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using).
+
+Google issues an ALTS credential for each service account running on the GCE VM.
```

Comment:
Instead of "Google issues an ALTS..", should this be "Google Cloud Platform issues an ALTS.."